components/nds-filters/__tests__/__snapshots__/nds-filters.test.js.snap

```diff
@@ -37,54 +37,55 @@ Array [
         </h2>
       </div>
```

Comment:
Please can you also update the readme with the new options? E.g. https://github.com/nice-digital/nice-design-system/tree/main/components/nds-filters#sorting